ballerina/tests/test_services.bal

```diff
@@ -1142,3 +1229,60 @@ service /nullable_inputs on basicListener {
         return account.number;
     }
 }
+
+service /subscriptions on basicListener {
+    isolated resource function get name() returns string {
+        return "Walter White";
+    }
+
+    resource function subscribe name() returns stream<string, error?> {
+        string[] names = ["Walter", "Skyler"];
+        return names.toStream();
+    }
+
+    isolated resource function subscribe messages() returns stream<int>|error {
+        int[] intArray = [1, 2, 3, 4, 5];
+        return intArray.toStream();
+    }
+
+    isolated resource function subscribe stringMessages() returns stream<string?, error?> {
+        string[] stringArray = ["1", "2", "3", "4", "5"];
```

Comment:
Since the resource returns a `string?` stream, shall we add a `null` value also?